FAQ-Improvements.md

```diff
@@ -0,0 +1,21 @@
+This is a list of "realworld" notes that have been encountered... so people
+can look at "cool new feature" and find what's known/been done before.
+
+1. "Let's seal it and use inductive charging!"
+  a)The Qi chargers are larger than the PETling so they have to be bent.
+		some Qi chargers get very unhappy about that (overheating/fire),
+		while others work but are *very* picky about placement on the Qi
+		charging pad.
+    There are inductive chargers that fit inside the cap (eg: 
+    https://amzn.to/3wslaLO , but they haven't been tried yet).
+
+  b)The seal for the PETling is at the top/inside; yeast and bacteria 
+    will get in the "threaded area" and will be impossible to wash out.
+    Right now, it's recommended to take the cap off and clean/sanitize
+    the cap and the PETling separately.
+
+2. Doesn't krausen/CO2 affect the readings?
```

Comment:
You can add a dryhopping as well.

Reply:
Looks good to me.  I dont' know about dryhopping...